apps/www/pages/_app.tsx

```diff
@@ -3,8 +3,7 @@ import "react-loading-skeleton/dist/skeleton.css";
 import type { AppProps } from "next/app";
 import { Inter, Roboto_Mono } from "next/font/google";
 import { Navbar } from "~/components/Navbar";
-import "@raystack/apsara/index.css";
-import "~/styles/styles.css";
+import "@raystack/apsara/style.css";
```

Comment:
Now we have a single css file for both the versions. Later we will simply remove the old css from the build.